{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "fbe9d838_84f886d6",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 5020
      },
      "writtenOn": "2023-03-07T06:21:11Z",
      "side": 1,
      "message": "This fails when running on Arm:\n\nLinux arm 6.0.0-0.deb11.6-cloud-arm64 #1 SMP Debian 6.0.12-1~bpo11+1 (2022-12-19) aarch64 GNU/Linux\n\n$ ./test_libaom --help\nfeenableexcept() failed",
      "revId": "9bf6e76925fc6c1b3a8a1161cc189bdc37693d41",
      "serverId": "e5514cf8-2d6e-3e29-adb4-24cd6dde4bf0"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "0bcaaba1_d797f099",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 9545
      },
      "writtenOn": "2023-03-07T17:36:07Z",
      "side": 1,
      "message": "Thank you for notifying it. I will look into this and consider reverting the CL.",
      "parentUuid": "fbe9d838_84f886d6",
      "revId": "9bf6e76925fc6c1b3a8a1161cc189bdc37693d41",
      "serverId": "e5514cf8-2d6e-3e29-adb4-24cd6dde4bf0"
    }
  ]
}